contrib/ansible/group_vars/all.yml

```diff
@@ -6,7 +6,7 @@ cluster_name: cluster.local
 # Account name of remote user. Ansible will use this user account to ssh into
 # the managed machines. The user must be able to use sudo without asking
 # for password unless ansible_sudo_pass is set
-#ansible_ssh_user: root
+ansible_ssh_user: fedora
```

Comment:
(Somewhat) unrelated should we add all.yml to .gitignore? It will make it harder to update all.yml in the repo, but things like this won't show up. I don't have inventory and all.yml on ignore, in case people wanted to check there changes into the repo. But maybe we should? 

(if we add it to .gitignore, making changes which get pushed public would require removing it, git add-ing the change, then putting it back in .gitignore)